### PR TITLE
Add scsbcul and scsbnypl locations #1134

### LINKED ISSUE #1134: Add scsbnypl and scsbcul locations in the templates
We need to update the locations template with the scsb locations. Specifically location_display.rb and locations.rb

A thought would be the code to be updated so that we generate
-  for scsbnypl code: recap$scsbnypl 
- for scsbcul code: recap$scsbcul 

per @kevinreiss The current file we receive from partners has a holding record as:
<marcxml:datafield tag="852" ind1="0" ind2="0">
      <marcxml:subfield code="h">PG8524.Z57</marcxml:subfield>
      <marcxml:subfield code="0">2119334</marcxml:subfield>
      <marcxml:subfield code="b">scsbcul</marcxml:subfield>
</marcxml:datafield>

I'm going through the code and will provide an update on this issue.

### PR DESCRIPTION
**Pull request by christinach**
closes #1134

### CODE REVIEW

**Comment by christinach** [comment]
Search results with scsb holding locations
Screen Shot 2021-03-23 at 10 54 23 PM

**Comment by christinach** [comment]
Search results with only Recap selected
Screen Shot 2021-03-23 at 10 54 46 PM

**Comment by christinach** [comment]
Advanced search with `scscbnypl` and `scsbcul` in the holding library
Screen Shot 2021-03-23 at 10 55 00 PM

**Comment by christinach** [comment]
Search results from scsb locations selected
Screen Shot 2021-03-23 at 10 55 22 PM

**Comment by christinach** [comment]
I deleted the previous screenshots. I'm adding new ones:
Search results with scsb
Screen Shot 2021-03-24 at 8 14 49 AM

**Comment by christinach** [comment]
Advanced search with scsb locations:
Screen Shot 2021-03-24 at 8 15 17 AM

**Comment by christinach** [comment]
advanced search result for scsbcul and nypl
Screen Shot 2021-03-24 at 8 16 07 AM

**Comment by christinach** [comment]
Record page of scsb. This is a long one.
Screen Shot 2021-03-24 at 8 17 28 AM

File: marc_to_solr/lib/princeton_marc.rb [hunk 1]
```diff
@@ -619,27 +619,38 @@ def alma_code?(code)
...
+  /^SCSB-\d+/.match?(record_id)
+end
+
 # holding block json hash keyed on mfhd id including location, library, call number, shelving title,
 # location note, location has, location has (current), indexes, and supplements
 # pulls from mfhd 852, 866, 867, and 868
 # assumes exactly 1 852 is present per mfhd (it saves the last 852 it finds)
 def process_holdings record # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
   all_holdings = {}
+  # extractor_doc_id = Traject::MarcExtractor.cached("001")
```
Comment (hackartisan) [inline marc_to_solr/lib/princeton_marc.rb:635]: can we delete the commented-out code?

**Review by hackartisan — CHANGES_REQUESTED** [review]
one small thing; otherwise this looks good to me!

It might be nice to have @mzelesky take a quick look at the screenshots.

File: marc_to_solr/lib/princeton_marc.rb (hunk 1 above)
Comment (christinach) [reply marc_to_solr/lib/princeton_marc.rb]: @hackartisan I removed them . Thanks

**Review by hackartisan — APPROVED** [review]

**Pull request merged**